scheduler/strategy/binpacking.go

```diff
@@ -54,5 +55,16 @@ func (p *BinPackingPlacementStrategy) PlaceContainer(config *dockerclient.Contai
 	// sort by highest weight
 	sort.Sort(sort.Reverse(weightedNodes))
 
-	return weightedNodes[0].Node, nil
+	topWeight := weightedNodes[0].Weight
+	for _, node := range weightedNodes {
```

Comment:
I think this could be written as:

```go
	for _, node := range weightedNodes; node.Weight == topWeight {
		node.Weight = int64(len(node.Node.Containers()))
		topNodes = append(topNodes, node)
	}
```

It will only iterate on top nodes (so if there are 100 nodes but only the first one has a better score it will stop immediately) and as added bonus it makes the code simpler.

WDYT?